### PR TITLE
{tools}[GCCcore/11.3.0,GCCcore/12.2.0] Scalene v1.5.26

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)
just backported to earlier toolchains to have it available

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
alvis-c1 - Linux Rocky Linux 8.8, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/64fa9f5e8b829667570f5958e0cd442f for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
@boegelbot please test @ jsc-zen2

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bwd-rockylinux-92 - Linux Rocky Linux 9.2 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.9.16
See https://gist.github.com/SebastianAchilles/f623ab64c45ff78521852c6e376377b5 for a full test report.

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @Micket!

File: easybuild/easyconfigs/s/Scalene/Scalene-1.5.20-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    ('pynvml', '11.4.1', {
+        'checksums': ['b2e4a33b80569d093b513f5804db0c7f40cfc86f15a013ae7a8e99c5e175d5dd'],
+    }),
+    ('commonmark', '0.9.1', {
+        'checksums': ['452f9dc859be7f06631ddcb328b6919c67984aca654e5fefb3914d54691aed60'],
+    }),
+    ('rich', '13.1.0', {
+        'checksums': ['81c73a30b144bbcdedc13f4ea0b6ffd7fdc3b0d3cc259a9402309c8e4aee1964'],
+    }),
+    ('astunparse', '1.6.3', {
```
Comment (smoors) [inline easybuild/easyconfigs/s/Scalene/Scalene-1.5.20-GCCcore-11.3.0.eb:36]: is this really needed? seems only required for Python < 3.9
https://github.com/plasma-umass/scalene/blob/2814d8c11b28ba480988eedbf1617af7a925c174/setup.py#L172C1-L172C54

Comment (Micket) [reply easybuild/easyconfigs/s/Scalene/Scalene-1.5.20-GCCcore-11.3.0.eb:36]: The setup.py requirements file was different in 1.5.20 release
https://github.com/plasma-umass/scalene/blob/033a9c6129ab6cc06d309e8bd3d5a8bd0d3fe3ba/setup.py#L205
but maybe they were wrong

**Pull request merged**